<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoClicker_widget</class>
 <widget class="QWidget" name="AutoClicker_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btn_1">
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/1.png</normaloff>:/red/1.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_2">
       <property name="icon">
        <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
         <normaloff>:/red/2.png</normaloff>:/red/2.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btn_clear">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lw_obj"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>btn_1</tabstop>
  <tabstop>btn_2</tabstop>
  <tabstop>btn_clear</tabstop>
  <tabstop>lw_obj</tabstop>
 </tabstops>
 <resources>
  <include location="../../../../../../lib2/icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
